README.rst

```diff
@@ -286,3 +286,7 @@ shell
     Config:
 
         container_name: Name of the Docker container.
+
+sync-media
+~~~~~~~~~~
+    Syncs the S3 media buckets between two namespaces (e.g. production to staging)
```

Comment:
The "S3" here makes it seem like this task is for AWS only. Is there a more generic word we could use?

Reply:
I've opted to just talk about them as generic `media buckets`. Seems common enough across both gcp and aws for object stores.